### PR TITLE
update vaccine flow URL

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
change URL based on VSP IA feedback

Page | New URL | Notes | Old URL
-- | -- | -- | --
New   covid appt | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment | Content uses the word "appointment", not "booking".  | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking
New   covid appt - contact facility | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/contact-facility | More accurate description of step - user will contact a single facility, not multiple | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-facilities
New   covid appt - dose history | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/confirm-doses-received | current URL (/received-dose) is a statement rather than a request for action or input | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/received-dose
New   covid appt - Choose facility | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/choose-facility | clear description of step in process | health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/facility
New   covid appt - Choose clinic | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/choose-clinic | clear description of step in process | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/clinic
New   covid appt - Select date | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/select-date |  | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/select-date-1
New   covid appt - Second dose  info | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/second-dose-info | More accurate description of step - step is providing info, not planning the next dose | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/plan-second-dose
New   covid appt - Contact info | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/contact-info | |/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-info
New   covid appt - Review | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/review | | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/review
New   covid appt - Confirmation | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-appointment/confirmation | | /health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/confirmation

## Testing done
local and unit test

## Screenshots
n/a

## Acceptance criteria
- [ ] All vaccine flow page URLs match IA guidance

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by LeKeve — APPROVED** [review]

File: src/applications/vaos/routes.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function createRoutesWithStore(store) {
             )}
           />
           <EnrolledRoute
-            path="/new-covid-19-vaccine-booking"
```
Comment (jbalboni) [inline src/applications/vaos/routes.jsx:46]: I think we still need to determine if we for sure need to redirect the old url, but wouldn't hurt to add it in regardless: https://reactrouter.com/web/api/Redirect

**Review by jbalboni — COMMENTED** [review]
One comment about redirecting the old url.

We'll have to be careful between this PR and my CTA changes to make sure we make the right updates, depending on which one is merged first.

File: src/applications/vaos/routes.jsx (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/routes.jsx:46]: @jbalboni, the files are ready to merge with master now. Did you want me to do something differently to the file?

Comment (jbalboni) [reply src/applications/vaos/routes.jsx:46]: @jenniemc We either need to confirm with Lauren that a redirect from the old urls is not needed, or go ahead and add a redirect in this PR.

Comment (jenniemc) [reply src/applications/vaos/routes.jsx:46]: @laurenernest, is the redirect from the Old URL address, `/new-covid-19-vaccine-booking` to the New URL address, `/new-covid-19-vaccine-appointment` required? 

cc @jbalboni

Comment (laurenernest) [reply src/applications/vaos/routes.jsx:46]: keep a redirect, but send to /health-care/schedule-view-va-appointments/appointments/new-appointment (similar to the non-covid pages)

**Pull request merged**